### ISSUE TITLE
using vimflowy from a smartphone?

### ISSUE THREAD

**Issue by mascip** [WuTheFWasThat/vimflowy#255]
Is it possible to use vimflowy from a Smartphone? It doesn't seem possible on mine.

That's the one thing missing for me, compared to workflowy which I can always access and modify from my pocket

**Comment by WuTheFWasThat** [comment]
yeah it's not good for that use case.  I know someone (see #218) is using vimflowy on their pixel, but I think they're using an external keyboard

**Comment by adamschmideg** [comment]
I tried it on my Android phone. The only key that works for me is Enter, so I can zoom into a node. But I can't switch to edit mode, pressing `i` or `o` does nothing.
